app.js

```diff
@@ -0,0 +1,353 @@
+
+        if (getTimeNow >= 5 && getTimeNow <= 11) {
+
+            greetDate.innerHTML = "Good Morning, ";
+            userNamePlace.innerHTML = userName + "!";
+
+        } else if (getTimeNow >= 12 && getTimeNow <= 18) {
+
+            greetDate.innerHTML = "Good Afternoon, ";
+            userNamePlace.innerHTML = userName + "!";
+
+        } else {
+
+            greetDate.innerHTML = "Good Evening, ";
+            userNamePlace.innerHTML = userName + "!";
+        }
+    }
+}());
+
+
+    //  set myGame object and store variables in it to use in functions
+var myGame = {
+    
+    myBody: document.getElementById('box-body-id'),
+    myBox: document.getElementById('box-id'),
+    playerTime: document.getElementById('playerTime'),
+    setTime: null,
+    endTime: null,
+    myTime: null,
+    flashOpt: document.getElementById('flashOpt')
+};
+    
+    //  Set a random color for the box or circle
+function setRandomColor() {
+    'use strict';
+
+    var letters = '0123456789ABCDEF'.split(''),
+        color = '#',
+        i;
+
+    for (i = 0; i < 6; i++) {
+        color += letters[Math.round(Math.random() * 15)];
+    }
+    return color;
+}
+
+    //  Not Working with "onclick" event in js file
+/*var gameLevelsSelect = {
+    levels: document.getElementById('levels-id').children,
+    easy: document.getElementById('easyOpt'),
+    medium: document.getElementById('mediumOpt'),
+    hard: document.getElementById('hardOpt'),
+    flash: document.getElementById('flashOpt')
+}
+gameLevelsSelect.levels.onclick = function () {
+    if (this[0]) {
+        gameLevelsFunctions.lvEasy();
+    }
+}*/
+    //  set levels functions
+
+function lvEasy() {
+    'use strict';
+    myGame.myBox.style.display = "none";
+    myGame.myBody.className = '';
+    myGame.myBody.className = 'easy';
+
+    //  Display a box or circle between 2 and 5 seconds
+    //  with a specific style 
+    var time = (Math.random() + 1) * 2500;
+    setTimeout(function () {
+
+        var moveTop = (Math.random() * 430),
+            moveLeft = (Math.random() * 540);
+
+        if (Math.random() > 0.5) {
+
+            myGame.myBox.style.borderRadius = "50%";
+
+        } else {
+
+            myGame.myBox.style.borderRadius = "0";
+        }
+
+        //  set the game box to easy style
+        myGame.myBox.className = '';
+        myGame.myBox.className = 'easy-box';
+        myGame.myBox.style.top = moveTop + 'px';
+        myGame.myBox.style.left = moveLeft + 'px';
+        myGame.myBox.style.backgroundColor = setRandomColor();
+        myGame.myBox.style.display = "block";
+        myGame.setTime = Date.now();
+    }, time);
+}
+
+function lvMedium() {
+    'use strict';
+    myGame.myBox.style.display = "none";
+    myGame.myBody.className = '';
+    myGame.myBody.className = 'medium';
+
+    //  Display a box or circle between 1 and 4 seconds
+    //  with a specific style
+    var time = Math.random() * 4000;
+    setTimeout(function () {
```

Comment:
You need to keep track of this variable, define a variable in the outer scope, then use `variablename = setTimeout( ... )` so you can later use `clearTimeout( variablename )` like you have in `lvHard`.